### DISCUSSION TITLE
[x][closed] experiment:tech_type

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#93] [Team Posts]
The wording confused me. Evidently you are expecting a type of assay (ChIP-Seq, RNA-Seq). To me technology means Illumina, Affymetrix...

**Comment by sveinugu** [comment]
assay_type?

**Comment by sveinugu** [comment]
Should actually include things like 'GWAS', 'Assembly annotation', 'Genome segmentation', and similar that are not really assays. So I thought of technology in a much broader sense than just sequencing technology. But agree we should choose something else to not confuse people.

**Comment by dzerbino** [comment]
result_type?

**Comment by sveinugu** [comment]
@dzerbino Result type seems to me to point towards the track file output, and does not seem precise enough. What about ‘technique’ or ‘method’?

**Comment by dzerbino** [comment]
OK, let's go with technique
